### PR TITLE
Add meeting minutes for the usergroup 2021-11-18

### PR DESCRIPTION
**Pull request by pfanguin**

### CODE REVIEW

File: content/pages/meeting_minutes/2021-11-18_de.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Title: Online, 18.11.2021
+Date: 2021-11-18
+Modified: 2021-11-18
+Authors: Guillaume REMY
+URL: meeting_minutes/2021-11-18
+Save_as: meeting_minutes/2021-11-18.html
+Slug: meeting_minutes/2021-11-18
+Lang: fr
```
Comment (pfanguin) [inline content/pages/meeting_minutes/2021-11-18_de.md]:
```suggestion
Slug: meeting_minutes/2021-11-18
Lang: de
```

**Review by pfanguin — COMMENTED** [review]
Thanks @svareg. 

Just a minor fix, otherwise LGTM !

**Comment by svareg** [comment]
Thanks for the Review @pfanguin , I made the change.
👍

**Review by rbovard — APPROVED** [review]
LGTM, please merge and deploy :)

**Pull request merged**